### ISSUE TITLE
Exception in bulk delete

### ISSUE THREAD

**Issue by richardolsson** [zetkin/organize.zetk.in#937]
Trying to delete several people using the bulk delete feature throws an unhandled promise rejection. Steps to reproduce:

1. Navigate to the people list
2. Select 3 person objects
3. From the bulk op list, pick "Delete 3 people" and click "Perform operation"

Expected results: The button should show a pending state, and the pane should then close with the people removed
Actual results: The button shows a pending state indefinitely. The console prints out an unhandled error warning.

**Comment by richardolsson** [comment]
Maybe this, @oelrich?

**Comment by richardolsson** [comment]
```
zetkinorganize_1     | Error executing bulk op person.delete
zetkinorganize_1     | Trace: TypeError: Cannot read property 'profile' of undefined
zetkinorganize_1     |     at req.z.resource.get.then.result (/var/app/build/app/server/api/bulk.js:139:89)
zetkinorganize_1     |     at process._tickCallback (internal/process/next_tick.js:68:7)
zetkinorganize_1     |     at exec.then.catch.err (/var/app/build/app/server/api/bulk.js:35:21)
zetkinorganize_1     |     at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by oelrich** [comment]
Yep!

**Comment by richardolsson** [comment]
When we discussed this earlier @oelrich, I hinted at another bug which allows a user to delete themselves from the organization and lose access. It seems this was fixed by pull request #984 a while ago, and I think that PR might be relevant to this issue as well. Maybe it introduced this issue?

**Comment by oelrich** [comment]
Is this also related to #1007? The error messages seem really similar.
